### PR TITLE
[docs] Code samples for accessors with subclassing && tidy up doc

### PR DESCRIPTION
**Pull request by katejeffreys**
Edits to Kevin's PR for no-wrap-accessor-docs

### CODE REVIEW

File: docs/_guide/properties.md [hunk 1]
```diff
@@ -410,31 +410,40 @@ get myProp() { ... /* Custom getter */ }
 this.myProp = 'hi'; // Invokes custom accessor
 ```
 
-If your class defines custom property accessors for a declared property directly on its prototype (i.e., not inherited from a superclass), LitElement will not overwrite them with generated accessors.  Such custom setters must manually request an update, supplying the property name and its old value to the update lifecycle by calling `requestUpdate`.
+If your class defines custom property accessors for a declared property directly on its prototype (i.e., not inherited from a superclass), LitElement will not overwrite them with generated accessors.
```
Comment (justinfagnani) [inline docs/_guide/properties.md]: Not sure "custom" is meaningful here. They're just standard JS accessors.

```suggestion
If your class defines its own accessors for a property, LitElement will not overwrite them with generated accessors. If your class does not define accessors for a property, LitElement will generate them, even if a superclass has defined the property or accessors.```

File: docs/_guide/properties.md [hunk 2]
```diff
@@ -410,31 +410,40 @@ get myProp() { ... /* Custom getter */ }
 this.myProp = 'hi'; // Invokes custom accessor
 ```
 
-If your class defines custom property accessors for a declared property directly on its prototype (i.e., not inherited from a superclass), LitElement will not overwrite them with generated accessors.  Such custom setters must manually request an update, supplying the property name and its old value to the update lifecycle by calling `requestUpdate`.
+If your class defines custom property accessors for a declared property directly on its prototype (i.e., not inherited from a superclass), LitElement will not overwrite them with generated accessors. 
+
+A custom setter must manually request an update, supplying the property name and its old value to the update lifecycle by calling `requestUpdate`.
```
Comment (justinfagnani) [inline docs/_guide/properties.md]:
```suggestion
The setters that LitElement generates automatically call `requestUpdate`. If you write your own setter you must call `requestUpdate` manually, supplying the property name and its old value.
```

File: docs/_includes/projects/properties/accessorssubclassing/my-element.js [hunk 3]
```diff
@@ -1,16 +1,39 @@
...
+  }
+
+  render() {
+    return html`<p>prop1: ${this.prop1}</p>
+
+    <button @click="${this.getNewVal}">change</button>
+  `;
+  }
+
+  updated(changedProperties) {
```
Comment (justinfagnani) [inline docs/_includes/projects/properties/accessorssubclassing/my-element.js]: Do we need these methods? I think it clouds the critical parts of the example and we could just have a getter and setter without any math or `update()`.

File: docs/_includes/projects/properties/accessorssubclassing/my-element.js [hunk 4]
```diff
@@ -1,16 +1,39 @@
-import SuperElement from './super-element.js';
+import { LitElement, html } from 'lit-element';
 
-class MyElement extends SuperElement {  
-  static get properties() { return {}; };
-  
-  get prop1() {
-    return this._prop1;
+export class MyElement extends LitElement {
```
Comment (justinfagnani) [inline docs/_includes/projects/properties/accessorssubclassing/my-element.js]: I think `SuperElement` was a good name

File: docs/_includes/projects/properties/customsetter/my-element.js [hunk 5]
```diff
@@ -2,37 +2,30 @@ import { LitElement, html } from 'lit-element';
...
-  set prop2(newVal) { this._prop2 = Math.floor(newVal); }
-  set prop3(newVal) {
-    let oldVal = this._prop3;
-    this._prop3 = Math.floor(newVal);
-    this.requestUpdate('prop3', oldVal);
+  // Custom setter for prop1
+  set prop1(newVal) {
+    let oldVal = this._prop1;
+    this._prop1 = Math.floor(newVal);
+    // Must
```
Comment (justinfagnani) [inline docs/_includes/projects/properties/customsetter/my-element.js]: unfinished?

**Comment by katejeffreys** [comment]
Ready for another look @kevinpschaaf @justinfagnani 

Staged: https://20190204t115009-dot-polymer-lit-element.appspot.com

**Comment by justinfagnani** [comment]
Comment on existing text. This: https://github.com/Polymer/lit-element/pull/489/files#diff-f37c02b7dd321fd8fca26553760681d7R16

Is a little vague. I'm not sure a reader unfamiliar with upgrades and this issue already will know what that's talking about. Maybe file an issue to clean up?

**Comment by katejeffreys** [comment]
> Comment on existing text. This: https://github.com/Polymer/lit-element/pull/489/files#diff-f37c02b7dd321fd8fca26553760681d7R16
> 
> Is a little vague. I'm not sure a reader unfamiliar with upgrades and this issue already will know what that's talking about. Maybe file an issue to clean up?

#528

**Pull request merged**